core/templates/pages/learner-dashboard-page/learner-dashboard-icons.component.html

```diff
@@ -45,7 +48,7 @@
     white-space: nowrap;
   }
   .oppia-dropdown-menu {
-    left: -150px;
+    left: -30vw !important;
```

Comment:
Thanks for raising this issue, @kevintab95 

@Radesh-kumar Can you please fix this issue? I'll approve this PR once it's resolved! :)

Reply:
I think we should use `!important` here since we are using ngbdropdown inline, somehow it is overriding the style.
(I've tried couple things using ngStlye and ngClass didn't work for me)
![Screenshot from 2021-05-31 00-16-29](https://user-images.githubusercontent.com/54100702/120116375-7ca6c180-c1a5-11eb-9acb-a53af13a49a4.png)

Reply:
Hi @Radesh-kumar, I took a look and it seems like using "dropdown-menu dropdown-menu-right" classes where "ngbDropdownMenu" is used will do the trick. With this you don't need the "left" CSS rule. Thanks!

Reply:
@kevintab95 done, thanks for looking into this.

Reply:
![image](https://user-images.githubusercontent.com/11008603/120192778-3d29b500-c239-11eb-8b2d-c12d6bb1e9a4.png)

Also, you can remove the class "oppia-dropdown-menu" as well, the remaining "top" rule seems to be applied by ngbDropdown dynamically (see screenshot).

Reply:
Thanks!

Reply:
@kevintab95 Thanks a lot for the help! :)